lens-doc/lens/private/syntax/srcloc.scrbl

```diff
@@ -0,0 +1,62 @@
+#lang scribble/manual
+
+@(require lens/private/doc-util/main)
+
+@title{Syntax object source locations}
+
+@defthing[syntax-srcloc-lens lens?]{
```

Comment:
We should use `lens/c` in documentation for new features. Updating the existing docs to use `lens/c` in more places has been on my todo list for a while.